src/Backend/Core/Engine/Base/Config.php

```diff
@@ -127,6 +122,22 @@ public function isActionAvailable(string $action): bool
         return false;
     }
 
+    public function isActionAvailableForActionType(string $action, string $actionType): bool
+    {
+        // If this action is disabled for this type, continue on to the next type
```

Comment:
Weird comment. It doesn't continue at all. It lets the caller know the given action is not available.